### ISSUE TITLE
docker default login

### ISSUE THREAD

**Issue by pocelka** [linkwarden/linkwarden#287]
**Is your question related to a problem or code? Please describe.**

I've spin up docker container with the v.2.0.0 and I've set `NEXT_PUBLIC_DISABLE_REGISTRATION` to `true`. What is the default login to get inside the application? I didn't find this anywhere in the documentation. Or do I need to first start container with false and then set it to true?

**Comment by SoCuul** [comment]
There's currently no default user created at login. You can either enable registration and create your account (then re-enable it), or directly add the account record to the database.

**Comment by pocelka** [comment]
Thank you for the answer. So basically you are saying that every user has the same role and there is not any kind of "administrator". Interesting concept - and I'm not saying that it is bad. I need to think about how to proceed.

Following the discussion, I think that it would be helpful to mention this in the documentation, which shouldn't be that big of a change. And at the same time somebody can tackle the copy & paste error (same settings for AWS? while there should be spaces* variables as per .env.sample):
![image](https://github.com/linkwarden/linkwarden/assets/20441822/086742f5-121a-477c-be4f-7c3a030ce05e)

**Comment by SoCuul** [comment]
> Thank you for the answer. So basically you are saying that every user has the same role and there is not any kind of "administrator". Interesting concept - and I'm not saying that it is bad. I need to think about how to proceed.
> 
> 
> 
> Following the discussion, I think that it would be helpful to mention this in the documentation, which shouldn't be that big of a change. And at the same time somebody can tackle the copy & paste error (same settings for AWS? while there should be spaces* variables as per .env.sample):
> 
> ![image](https://github.com/linkwarden/linkwarden/assets/20441822/086742f5-121a-477c-be4f-7c3a030ce05e)
> 
> 

Yup, every user has the same permissions. I think they're working on creating an administrator account implementation, but I may be wrong.

**Comment by daniel31x13** [comment]
Hey there, a lot has changed. Has this been resolved? Will reopen this issue if it hasn't.
